modules/checklists/checklists.php

```diff
@@ -772,7 +772,7 @@ public function save_post_meta_box($id, $post)
             // Authentication checks: make sure data came from our meta box and that the current user is allowed to edit the post
             // TODO: switch to using check_admin_referrer? See core (e.g. edit.php) for usage
             if (!isset($_POST[self::METADATA_TAXONOMY . "_nonce"])
-                || !wp_verify_nonce($_POST[self::METADATA_TAXONOMY . "_nonce"], __FILE__)) {
+                || !wp_verify_nonce(sanitize_text_field($_POST[self::METADATA_TAXONOMY . "_nonce"]), __FILE__)) {
```

Comment:
Nonce can be sanitized using `sanitize_key`

Reply:
updated